ajax/libs/schemapack/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "schemapack",
+  "filename": "schemapack.min.js",
+  "version": "1.4.2",
+  "description": "The fastest and smallest JavaScript object serialization library. Efficiently encode your objects in to compact byte buffers and then decode them back in to objects on the receiver. Integrates very well with WebSockets.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/phretaddin/schemapack.git"
+  },
+  "keywords": [
+    "schema",
+    "pack",
+    "binary",
+    "websocket",
+    "serialize",
+    "serialization",
+    "schemapack",
+    "bytebuffer",
+    "buffer",
+    "arraybuffer",
+    "net"
+  ],
+  "author": "phretaddin",
+  "license": "MIT",
+  "npmName": "schemapack",
+  "npmFileMap": [
+    {
+      "basePath": "build",
```

Comment:
@sajochiu 
There is no `build` folder and `schemapack.min.js` in npm package.
Please modify the filemap and manually minify the main file.